### DISCUSSION TITLE
How to limit prints quantity

### DISCUSSION THREAD

**Discussion by rektosaure** [photobooth-app/photobooth-app#294] [Q&A]
Hi,
Is there an option to limit the number of print available ?
And the way to reset it? Something similar to stats?

**Comment by mgineer85** [comment]
There is no such a functionality built in.
You could write a bash script checking a limit and trigger the print if within limit.

**Comment by rektosaure** [comment]
It would be great to build it if possible.
Additionally, it would be awesome to have an option to display the number of already printed pictures in the gallery area.

**Reply by mgineer85** [reply]
Feel free to send a feature request. Maybe someone has time to implement at one point.
